content/tutorials/themes/theme-objects/text/readme.md

```diff
@@ -0,0 +1,114 @@
+---
+uid: text  
+locale: en  
+title: TEXT Theme object  
+dnnversion: 09.02.00  
+previous-topic: ddrmenu  
+next-topic: theme-objects
+related-topics: theme-objects,themes,create-theme  
+links:  
+---
+
+# TEXT Theme Object Introduction  
+
+Displays Localized text in your theme / skin, also supports the use of DNN Core Token replacement. 
+ 
+The text will be loaded from a resource file (*.resx), you need a separate resx file per language. 
+ 
+These should be located in a folder named "App_LocalResources" (in the Theme folder). 
+ 
+  
+ 
+## Naming convention: 
+ 
+When your skin file = index.ascx: 
+Default language: index.ascx.resx 
+French: index.ascx.fr-FR.resx 
+German: index.ascx.de-DE.resx 
```

Comment:
I tried this on my test theme and with 
![image](https://github.com/DNNCommunity/DNNDocs/assets/8999382/1d13d256-8725-47f1-a023-7387be7cae83)
it does not work.
It does work this though: 
![image](https://github.com/DNNCommunity/DNNDocs/assets/8999382/f4882e43-72b3-496f-886a-9fb7979258f6)

Reply:
https://github.com/dnnsoftware/Dnn.Platform/blob/9367336b703f8251c4c9c09fa09655246cd10513/DNN%20Platform/Website/admin/Skins/Text.ascx.cs#L32

Reply:
@valadas I've heard you say this before on the community forums, but I have yet to see this in practice (just haven't ever personally used this naming convention or seen it work).  

I'm with @Timo-Breumelhof here as far as the docs go, because this has always been the convention.  Perhaps there is some newer shortcut naming convention for RESX files in this context, but based on the referenced line of code, it is dependent on the value `PortalSettings.ActiveTab.SkinSrc`, which is by default something like `[G]Skins/MyThemeName/my-theme-layout.ascx`.  So the resulting file being looked for would include `.ascx`.